docs/DeveloperGuide.md

```diff
@@ -201,6 +201,58 @@ There are multiple overloaded methods:
 1. archiveNotes(): is used to archive note by integer or note title.
 1. unarchiveNotes(): is used to unarchive note by integer or note title.
 
+The rationale for overloading such methods are given below:<br>
+
+**1. getNote()**<br>
+- There are a total of 3 getNote() methods.
+    1. First one takes in an `int` index as an argument.
+    2. Second takes in a `String` of the note title.
+    3. The third takes in a `String` of the note title, and a `boolean` isArchive.  
+
+- getNote(`int`) returns the Note that is in the position of specified index within the default notebook.
+- getNote(`String`) checks if the note of the specified title exists in the default notebook and returns a `boolean` value.
+- getNote(`String`, `boolean`) checks if the note of the specified title exists and returns a `boolean` value. The `boolean` acts as a flag to determine which of the notebook (default/archived), will be streamed and filtered.
```

Comment:
Use list of notes instead of notebook since there is only 1 notebook but 2 arraylist of notes. Using default/archive notebook may give a wrong idea.
```suggestion
- getNote(`String`) checks if the note of the specified title exists in the notebook's default list of notes and returns a `boolean` value.
- getNote(`String`, `boolean`) checks if the note of the specified title exists and returns a `boolean` value. The `boolean` acts as a flag to determine which of the list of notes (default/archived) in the notebook, will be streamed and filtered.
```

Reply:
Likewise for all the description below.